index.rst

```diff
@@ -1,61 +1,546 @@
+        used as well. Old configuration files can be kept in the repo if they
+        still represent valid configurations otherwise, they should be removed.
+        Note, though, that they will still remain available on previous
+        versions in the git repo, enabling historical comparison.
+
+        Alternatively, load the new configuration into the configuration
+        database, if that is the case for the CSC in question.
+
+    #.  Modify the configuration labels so that it maps to the new
+        configuration (preferred) or create a new label for the new
+        configuration. For Salobj CSC, this is done by editing the
+        ``_labels.yaml`` file.
+
+    #.  Commit the changes made to the configuration and push it to the
+        online repository.
+
+    #.  Testing and verification of the new configuration.
+
+        The complexity of this step will also vary considerably from component
+        to component. In some cases, it might be feasible to test the
+        configuration in the actual running CSC. For instance, in the filter
+        swap use-cases, one can simply pull the ticket branch in the CSC host
+        for testing, and exercise the CSC to verify it is working.
+
+        In other situations the new configuration can be tested and verified
+        using simulators.
+
+        Nevertheless, in some other more critical cases tests and verification
+        are only possible/feasible with on-sky operations.
+
+        Regardless of which case the CSC falls into, the approach should be to
+        run the test and verification with the repository ticket branch. If any
+        issues arises during this test the process can either continue to the
+        next phase or start over. The Jira ticket created to track the work
+        should reflect the results of any tests.
+
+    #.  Create/update related documentation.
+
+
+    #.  Create pull requests (PRs). Once the configuration is tested, verified
+        and documented, it is ready to be officially accepted.
+
+        PRs must be created for all repositories that where modified during
+        the process, including, but not limited to, the configuration
+        repository, ancillary software and documentation. The PRs will follow
+        the standard review procedure. Once the they are approved, merged and
+        released the new configuration becomes official and can be deployed.
 
-.. note::
 
-   **This technote is not yet published.**
+During commissioning we anticipate that there may be situations where quick
+configuration changes may need to be implemented. In these cases, the user
+should also create a Jira ticket (or work out of an existing ticket) to
+document the occurrence. Then, instead of checking out the repository
+locally, the user can work out of the deployed CSC configuration directly
+in the host. It is important to create a branch in place to work on and,
+later, commit-push to the repository and continue with the process
```

Comment:
Push access from within containers is troublesome without the container having credentials present. That's not a road I'd want to go down. I think the language we could use here is that we would have to map a physically located repo onto the container's config repo location. Then the above statements work.

Reply:
I'd rather not go into implementation details, but I can make a note about it here...